packages/component-library/components/Heading/Heading.module.scss

```diff
@@ -63,3 +63,42 @@
   line-height: $kz-typography-heading-6-line-height;
   letter-spacing: $kz-typography-heading-6-letter-spacing;
 }
+
```

Comment:
Since the `color` property has been decoupled from the heading classes, my preference would be to remove `color` from `.heading` (line 7). If a user does not pass in a colour it defaults to `dark` so this is essentially dead code.

Reply:
👍